### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by dasileker**
## restaurant-page
 This project comes after the introduction of the ES6 class keyword, ES6 modules, and Webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using Webpack.

## Built With

 - Only Javascript 
 - Webpack
 - Used the DOM Manipulation.
 - CSS
 - Images
 - Linters

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { getInnerHTML } from 'domutils';
+import Navigation from './navigation';
+import initPage from './pages/mainpage';
+import homePage from './pages/home';
+import aboutPage from './pages/about';
+import menuPage from './pages/menu';
+import contactPage from './pages/contact';
+// import favicon from '/src/assets/burger-favicon.png';
+// import "./main.css";
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!! :muscle:

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+function html() {
+  const link = document.querySelector("link[rel*='icon']")
+    || document.createElement('link');
+  link.type = 'image/x-icon';
+  link.rel = 'shortcut icon';
+  // link.href = favicon;
```
Comment (addod19) [inline src/index.js]: Kindly remove commented code, use git history instead!! :muscle:

File: src/pages/about.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// import hero from '/src/assets/about-photo';
```
Comment (addod19) [inline src/pages/about.js]: Kindly remove commented code, use git history instead!! :muscle:

File: src/pages/contact.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// import PhoneNumber from "/src/assets/phone-number.png";
+// import Email from "/src/assets/Email.png";
```
Comment (addod19) [inline src/pages/contact.js]: Kindly remove commented code, use git history instead!! :muscle:

File: src/pages/mainpage.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+// import logo from "/src/assets/Logo.png";
```
Comment (addod19) [inline src/pages/mainpage.js]: Kindly remove commented code, use git history instead!! :muscle:

File: src/pages/menu.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
+// import { ThiccBurger } from './src/assets/thicc-bruger.png';
+// import RedBurger from '/src/assets/red-burger.png';
+// import VeryThiccBurger from '/src/assets/very-thicc-burger.png';
+// import ThiccFries from '/src/assets/thicc-fries.png';
```
Comment (addod19) [inline src/pages/menu.js]: Kindly remove commented code, use git history instead!! :muscle:

File: package.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "This is my first time using webpack.  It's complicated and needs a lot of time to get used to.",
+  "private": true,
+  "scripts": {
+    "start": "webpack serve --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js"
+  },
```
Comment (addod19) [inline package.json]: Webpack is not properly set up, currently, we have no webpack.dev.js or webpack.prod.js file. Hence, the `npm run start/build` command returns with an error :bug: Let us kindly fix it. Or we could simple use `"dev": "webpack --mode development",
    "build": "webpack --mode production",
`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @dasileker, great job :1st_place_medal: THe README is professional, commit messages easy to understand, PR has a title and a brief description :fireworks: Awesome. Let us kindly fix the comments below in order to get the project approved :smile: 
- Kindly fix the live liNK in the README
- Kindly fix the linter errors
- Kindly use ES6 arrow functions
- Kindly remove commented code, use git history instead

Once these are resolved, kindly submit another code review request
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by dasileker** [comment]
hello, the linters is good, but the  errors is from the deployment  ot from the linters

File: package.json (hunk 7 above)
Comment (dasileker) [reply package.json]: okay, sorry I did change the Webpack I take just the  Webpack.config.js only

File: src/pages/menu.js (hunk 6 above)
Comment (dasileker) [reply src/pages/menu.js]: done.

File: src/pages/mainpage.js (hunk 5 above)
Comment (dasileker) [reply src/pages/mainpage.js]: done

File: src/pages/contact.js (hunk 4 above)
Comment (dasileker) [reply src/pages/contact.js]: done.

File: src/index.js (hunk 2 above)
Comment (dasileker) [reply src/index.js]: done

File: src/index.js (hunk 1 above)
Comment (dasileker) [reply src/index.js]: done.

File: src/pages/about.js (hunk 3 above)
Comment (dasileker) [reply src/pages/about.js]: done.

File: README.md [hunk 8]
```diff
@@ -1 +1,54 @@
...
+
+  - To Make the website fully working make sure to install the webpack
+  and run the folowing command if yoy don't have it.
+
+  - npm install webpack -g "to make the installation global".
+
+  - npm install webpack-cli --save-dev.
+  - add this comand if you want to work with the webpack server " npm install --save-dev webpack-dev-server.
+
+  - voila Now youcan view the page in your localhost ENV.
```
Comment (christianotieno) [inline README.md]: ## Improve  the README
- [x] I feel these instructions may not be necessary. I would suggest we remove them and only add the features of running the webapck in the `package.json` file using the `start` script. What do you think?

File: package.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "reastaurant-page for Microverse",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (christianotieno) [inline package.json]: ## Add missing script
- [x] Kindly add back the `start` script here.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @dasileker 👋 
Great improvements from the last review.
You seem to have removed the `start` scripts in your series of changes. Why is that so?
### Happy coding 💻 
#### Christian Otieno (CR)

File: package.json (hunk 7 above)
Comment (dasileker) [reply package.json]: done.

File: package.json (hunk 9 above)
Comment (dasileker) [reply package.json]: ok. done I did return back the start scrip with the:  
```
"scripts": { "start": "webpack serve --config webpack.dev.js", 
 "build": "webpack --config webpack.prod.js"
 },
```

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Comment by dasileker** [comment]
thanks, to All the TSE's

**Pull request merged**